src/Utility/LagrangeInterpolator.C

```diff
@@ -29,7 +29,7 @@ Matrix LagrangeInterpolator::get(const std::vector<double>& new_grid)
   Matrix result(grid.size(), new_grid.size());
   for (size_t i = 0; i < grid.size(); ++i)
     for (size_t j = 0; j < new_grid.size(); ++j)
-      result(i+1, j+1) =  Lagrange(new_grid[j], i, grid);
+      result(j+1, i+1) =  Lagrange(new_grid[j], i, grid);
```

Comment:
Sure about this? If so, it looks like the dimensioning of the matrix is wrong too, i.e., it should be:
`Matrix result(new_grid.size(),grid.size());`

Reply:
yeah, it should be sized like that as well. it's one row per point, one column per basis function. i screwed up converting code using reversed row/column syntax for blas compatibility..